### DISCUSSION TITLE
Remove large files from the repo

### DISCUSSION THREAD

**Discussion by jinincarnate** [gitextensions/gitextensions#10611] [Q&A]
I want to clean my repo lfs storage, what are the steps for the same.

**Comment by pmiossec** [comment]
GitExtensions doesn't have feature to handle git lfs so you will have to use the command line.

Is that `git lfs prune` that you are looking for?

**Comment by jinincarnate** [comment]
actally there is an option 
![image](https://user-images.githubusercontent.com/9129660/210834750-babf142e-0165-4396-b7cd-2a592c834552.png)
which lists all the large files and lets u delete them. But i m not sure how exactly this features works or what are the steps.
i have tried identifying all the large files and deleting all the old versions of a particular file and force pushing. but can still see those large files when i try to list them again

**Reply by pmiossec** [reply]
> actally there is an option

Yes but that's absolutely not linked to lfs as you asked before........ 

>but can still see those large files when i try to list them again

After changing the history, you have to git gc but expire reflog first.
See last paragraph there : https://www.git-scm.com/docs/git-filter-branch/2.2.3#_checklist_for_shrinking_a_repository
